app/config.js

```diff
@@ -52,5 +67,10 @@ module.exports = require('rc')('agregore', {
   // All options here: https://github.com/webtorrent/webtorrent/blob/master/docs/api.md
   btOptions: {
     storageLocation: DEFAULT_BT_DIR
+  },
+  // All options here: https://gun.eco/docs/API#-a-name-gun-a-gun-options-
+  gunOptions: {
+    peers: DEFAULT_GUN_PEERS,
```

Comment:
Would ti be possible to tell GUN to store data inside `path.join(USER_DATA, 'gun')`?

Reply:
looking into it, talking with gundb folks. gundb wants to write only to it's own directory, i am trying to see if we can save the data in other folders.

Reply:
Is there a global folder that all gundb instances use?

Reply:
to use a folder we can pass in the folder using the file option, Gun({file: 'somefolder'}). so i will make those changes. i would think that gundb shouldn't use a global folder, since gundb uses radix tree in those files, i think mixing those files will mess gundb up. the radix stuff is beyond me, but from reading their docs, it seems like the files are gonna need to stay untouched and not mixed up.

there is another thing, while we can use our own folder using the 'file' option. there will be a error that will fire every 5 seconds. it will be a silent error, it will not crash or kill the module.

https://github.com/amark/gun/blob/master/lib/stats.js#L65 <---- that is the line that creates the error. on the console every 5 seconds, you will see this
`Error: ENOENT: no such file or directory, open 'C:\Project\gundb\tests\node_modules\gun\stats.C:\Project\gundb\someFolderHere'`
i made a pull request to the gundb repo to fix this. so if you don't mind this small error, then we are good to go after i make the changes. but if you have other thoughts, then let me know.

Reply:
Hmm, it might be annoying to folks. What about using the [mkdirp](https://www.npmjs.com/package/mkdirp) module to ensure the directory exists within the gun-protocol handler when it starts up. 🤔

Reply:
it wants to write to the node_modules/gun folder. so the directory for sure exists, the error is happening because of this `__dirname+'/../stats.'+(opt.file || 'radata')` opt.file is our custom directory where we wanna store the gundb files. if our directory that we pass is 'C:\Project\gundb\someFolderHere' then it will try to create a file with the filename being 'stats.C:\Project\gundb\someFolderHere', so that becomes `node_modules/gun/stats.C:\Project\gundb\someFolderHere` i think the problem is fs.writeFile is having trouble with the filename.

i pasted my pull request on their matrix chat, hopefully, they get to it.